packages/flutter/test/widget/bottom_sheet_test.dart

```diff
@@ -23,6 +23,7 @@ void main() {
     await tester.pump();
     expect(find.text('BottomSheet'), findsNothing);
 
+    showBottomSheetThenCalled = false;
```

Comment:
Why separate the declaration and initialization?  If you prefer, you can move the declaration here too.